requirements-dev.txt

```diff
@@ -28,3 +28,4 @@ python-semantic-release==7.3.0
 requests-mock==1.8.0
 reuse==0.12.1
 syrupy==0.8.4
+django-watchman==1.2.0
```

Comment:
This should go into `requirements.txt`. And we try to keep our requirements files in alphabetical order.